src/status_im/contexts/wallet/collectible/events.cljs

```diff
@@ -187,24 +187,31 @@
 (rf/reg-event-fx :wallet/trigger-share-collectible
  (fn [_ [{:keys [title uri]}]]
    {:fx [[:effects.share/open
-          {:title   title
-           :message title
-           :url     uri}]]}))
+          {:options
+           {:title   title
+            :subject title
+            :message uri}}]]}))
 
 (rf/reg-event-fx :wallet/share-collectible
```

Comment:
I think that `in-sheet?` as an argument adds unnecessary knowledge about the outer context to this event. Instead we can pass `dispatch-timeout` and make decision of selecting `dispatch` vs `dispatch-later` based on it. And `:hide-bottom-sheet` could be called outside, where knowledge about UI details exist. This will make `:wallet/share-collectible` more flexible.
Wdyt?

Reply:
yes, that's a good point! will adjust to that!

Reply:
hmm on reviewing this, I think one thing to consider is that `in-sheet?` will hide the bottom sheet. 
But anyway I see there is only one use of this function and it's in a bottom sheet, so I'll just remove that parameter altogether :)